bakery_cli/scripts/genmetadata.py

```diff
@@ -453,47 +454,32 @@ def getSize(familydir):
     return int(gzipSize)
 
 
-def setIfNotPresent(metadata, key, value):
-    if key not in metadata:
-        metadata[key] = value
-
-
 def genmetadata(familydir):
-    metadata = InsertOrderedDict()
     if hasMetadata(familydir):
         metadata = loadMetadata(familydir)
-        print(metadata)
-    familyname = inferFamilyName(familydir)
-
-    if not metadata.get('name') or metadata.get('name', "UNKNOWN") == "UNKNOWN":
-        metadata["name"] = familyname
-
-    desName = getDesigner(familydir)
-    if not metadata.get('designer'):
-        metadata["designer"] = desName
-                    # DC Should check it against profiles.json
-
-    if not metadata.get('license'):
-        metadata["license"] = inferLicense(familydir)
-
-    setIfNotPresent(metadata, "visibility", "Sandbox")
-
-    category = ''
-    if check_monospace(familydir):
-        category = 'monospace'
-    setIfNotPresent(metadata, "category", category)
-                    # DC Should get this from the font or prompt?
-
-    if not metadata.get('size') or metadata.get('size', -1) == -1:
-        metadata["size"] = getSize(familydir)
-                    # DC: this should check the filesize got smaller than last
-                    # time
-    metadata["fonts"] = createFonts(familydir, familyname)
-    metadata["subsets"] = inferSubsets(familydir)
-    setIfNotPresent(metadata, "dateAdded", getToday())
-                    # DC This is used for the Date Added sort in the GWF
-                    # Directory - DC to check all existing values in hg repo
-                    # are correct
+    else:
+        metadata = fonts_pb2.FamilyProto()
+        metadata.name = inferFamilyName(familydir)
+        metadata.designer = getDesigner(familydir) # DC Should check it against profiles.json
+        metadata.license = inferLicense(familydir)
+
+        # DC Should get this from the font or prompt?
+        if check_monospace(familydir):
+            metadata.category = 'monospace'
+        else:
+            metadata.category = ''
+
+        # DC: this should check the filesize got smaller than last time
+        # FS: this field was deprecated in the protobuf messages
+        # metadata.size = getSize(familydir)
```

Comment:
May I make another commit completely removing this metadata.size comment and the getSize function ?